src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -78,11 +78,31 @@ private void validateFullUpdateDetails(OverseasEntitySubmissionDto overseasEntit
         // ownersAndOfficersDataBlockValidator.validateOwnersAndOfficers(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
We will need to activate this line as we have now implemented the statements ?

Reply:
separate story for that (uar-711)